app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Users
+  class OmniauthCallbacksController < Devise::OmniauthCallbacksController
+    # You should configure your model like this:
+    # devise :omniauthable, omniauth_providers: [:twitter]
```

Comment:
Resolved this in a meeting with Rob on Tuesday - I will focus on just content of the current commits for future code reviews, not proposals for further work.